setup.py

```diff
@@ -3,7 +3,7 @@
 import os.path
 
 package_name = "dbt"
-package_version = "0.5.2"
+package_version = "0.5.3"
```

Comment:
sorry, `development` was out of date with `master`. If you merge `development` back into `integration-suite`, then this should go away